spectral_cube/lower_dimensional_structures.py

```diff
@@ -417,10 +417,12 @@ def quicklook(self, filename=None, use_aplpy=True, aplpy_kwargs={}):
                                                        **aplpy_kwargs)
 
                 self.FITSFigure.show_grayscale()
-                self.FITSFigure.add_colorbar()
+                if not hasattr(self, 'colorbar'):
+                    self.FITSFigure.add_colorbar()
                 if filename is not None:
                     self.FITSFigure.save(filename)
-            except (wcs.InconsistentAxisTypesError, ImportError):
+            except Exception as e:
+                print("Encountered errors when creating image:", e)
```

Comment:
Could you remove this change from this PR?  It's not related to the main issue here, so we'd like to have it in a separate PR